### ISSUE TITLE
Tabs read twice using Talkback on Android

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6488]
## Environment

> Operating system

Android Pie

> Browser

DuckDuckGo/Firefox Mobile

> Assistive technology used to verify

Talkback

## Detailed description

> What version of the Carbon Design System are you using?

10.15.0

> What did you expect to happen?

Tabs should be read once

> What happened instead?

Tabs were read twice "tab label two, tab; double tap to activate." (swipe gesture) "tab label two, link; double tap to activate" 

## Additional information

ref #6410

**Comment by dakahn** [comment]
couldn't reproduce using latest tabs
